### DISCUSSION TITLE
Upgrade Navidrome from v0.51.1 to v0.52.5

### DISCUSSION THREAD

**Discussion by novellahub** [tteck/Proxmox#3189] [General] [answered]
Is there a way to upgrade Navridrome from from v0.51.1 to v0.52.5?  I have done apt update && apt upgrade in the LXC container.  Or do I need to download the binary and install manually?

**Comment by tteck** [comment, marked as answer]
![Screenshot 2024-06-13 9 44 34 AM](https://github.com/tteck/Proxmox/assets/21342670/4a138e3e-b519-493b-bbc6-902dd9f609e7)
